_posts/2016/2016-04-20-jenadevs-docker-party.md

```diff
@@ -0,0 +1,72 @@
+We had a demo on the basic commands along with some live coding on the terminal, where several containers were also inspected at runtime.
+The second part considered some best practises for writing your own Dockerfile, which was based on the [official Docker recommendations](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices), tips by [Michael](http://crosbymichael.com/dockerfile-best-practices.html) [Crosby](http://crosbymichael.com/dockerfile-best-practices-take-2.html) as well as the experiences of [epagesdevs](http://twitter.com/epagesdevs) with developing, testing, integrating, deploying and running supportive images for our eCommerce platform.
+This was a nice opportunity to also show some of our typical use cases at ePages, e.g. in the Continuous Delivery Pipeline, for integration testingor in the operations area of our own and provider infrastructure based already partly on [Kubeneretes](http://kubernetes.io).
+The talk finished with some typical base images as well as a discussion about official app environment images.
+
+## Containers, Containers, Con... er Workshops!
+
+Upfront, all necessary material and code for the workshops was hosted on the jenadevs [GitHub](https://github.com/jenadevs) account, which even offers the [complete presentation](https://github.com/jenadevs/jenadevs-meetup-001-docker-party).
+Great news: _All material of **[jenadevs](https://github.com/jenadevs)** is free and will always be open-source!_
+
+### Beginners Workshop
+
+The goal of the beginner workshop was to setup the Docker environment, learn the basic commands, run the workflow and become familiar with Docker Compose.
+The participants followed the [official Docker birthday tutorial](https://github.com/docker/docker-birthday-3/blob/master/tutorial.md), while the more experienced Docker users, Benjamin Nothdurft and Kay Abendroth, offered assistance.
+As kittens bustled around the beamer and laptop screens, the Docker newbies built simple web applications using the Python microframework [Flask](http://flask.pocoo.org/) – all running in a container, of course.
+
+### Advanced Workshop
+
+This workshop focused on writing your own Dockerfiles, run the containers and integration testing with [CircleCI](https://circleci.com). Bastian Klein prepared an empty custom [Magento](https://magento.com) `Dockerfile` and `Circle.yml` with comments left only, where the implementation code was completely removed.
+The attendes then had to figure out how the Dockerfile commands needed to be configured under the guidance of the workshop maintainer.
+
+### Special Workshop
+
+The most experienced Dockers gathered together in the special workshop, where the main focus was a bit shifted. We had prepared three worstation laptops. Andreas Grohman introduced CoreOS to the each participant. The goal was to setup a cluster environment with three nodes and run Docker on it.
```

Comment:
The most experienced Docker users...
Three workstation laptops had been prepared to introduce CoreOS to the participants.
...and to run Docker on it.